### ISSUE TITLE
I got a nan loss on vocoder train

### ISSUE THREAD

**Issue by fabianoluzbr** [CorentinJ/Real-Time-Voice-Cloning#797]
Hi folks,

I've already made changes to my learning rate, I've already changed the batch size and it didn't. 

`Found 117946 samples
+------------+--------+--------------+
| Batch size |   LR   | Sequence Len |
+------------+--------+--------------+
|    100     | 0.0001 |     1000     |
+------------+--------+--------------+
 
{| Epoch: 1 (989/1180) | Loss: nan | 2.3 steps/s | Step: 0k | }
`
does anyone know what this might be?

**Comment by blue-fish** [comment]
Maybe your spectrogram or wav data contains a NaN.

**Comment by Bebaam** [comment]
Just for the record: I also had this problem. 
It occured for vocoder training only, as there were NaN values in some files in mels_gta. 
Deleting corresponding files in folders audio, embeds, mels and mels_gta as well as in synthesized.txt solved that error for me.
